content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewers:
+- msau42
+- xing-yang
+title: Volume Attributes Classes
+content_type: concept
+weight: 40
+---
+<!-- overview -->
+
+This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
+[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+in Kubernetes.
+
+<!-- body -->
+
+## Introduction
+
+A VolumeAttributesClass provides a way for administrators to describe the mutable
+"classes" of storage they offer. Different classes might map to different quality-of-service levels.
+Kubernetes itself is unopinionated about what these classes represent.
+
+
+## The VolumeAttributesClass Resource
+
+Each VolumeAttributesClass contains the `driverName` and `parameters`, which are used when a PersistentVolume belonging to the class needs to be dynamically provisioned or modified.
+
+The name of a VolumeAttributesClass object is significant and is how users can request a particular class.
+Administrators set the name and other parameters of a class when first creating VolumeAttributesClass objects.
+While the name of a VolumeAttributesClass object in a PersistentVolumeClaim is mutable, the parameters in an existing class are immutable.
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: silver
+driverName: pd.csi.storage.gke.io
+parameters:
+  provisioned-iops: "3000"
+  provisioned-throughput: "50" 
+```
+
+
+### Provisioner
+
+Each VolumeAttributes has a provisioner that determines what volume plugin is used for provisioning PVs. This field must be specified. 
+
+The feature support for VolumeAttributesClass is implemented in [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner).
+
+You are not restricted to specifying the [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner). You can also run and specify external provisioners,
+which are independent programs that follow a specification defined by Kubernetes.
+Authors of external provisioners have full discretion over where their code lives, how
+the provisioner is shipped, how it needs to be run, what volume plugin it uses (including Flex), etc. 
```

Comment:
Can you remove " (including Flex)"?  Flex volume is already deprecated so we don't want to mention it as an example.